Sources/ValueType.swift

```diff
@@ -49,14 +49,39 @@ extension Int64: ValueType {
 }
 
 extension Array where Element: ValueType {
-    public static func value(from object: Any) throws -> [Element] {
+    public static func value(from object: Any, discardInvalidObjects: Bool = false) throws -> [Element] {
```

Comment:
K. `discardingErrors` it is. @jgrandelli if you would modify this when you have some spare time, I'll get this all merged. Thanks!